app.js

```diff
@@ -2420,6 +2568,56 @@ define(function(require) {
 					args.hasOwnProperty('error') && args.error(parsedError);
 				}
 			});
+		},
+
+		accountsUpdateSecurity: function(data, callback) {
+			var self = this;
+
+			self.callApi({
+				resource: 'security.update',
+				data: {
+					accountId: data.accountId,
+					data: data.data
+				},
+				success: function(data) {
+					callback && callback(data.data);
+				}
+			});
+		},
+
+		accountsGetPhoneNumbers: function(data, callback) {
+			var self = this;
+
+			self.callApi({
+				resource: 'numbers.list',
+				data: {
+					accountId: data.accountId,
+					filters: {
+						pagiante: false
+					}
+				},
+				success: function(data) {
+					callback && callback(data.data);
+				}
+			});
+		},
+
+		accountsGetOomaSMSBoxes: function(data, callback) {
+			var self = this;
+
+			self.callApi({
+				resource: 'oomasmsboxes.list',
+				data: {
+					accountId: data.accountId,
+					filters: {
+						paginate: false
+					},
+					generateError: false
+				},
+				success: function(data) {
+					callback && callback(data.data);
+				}
+			});
```

Comment:
What should happen if this API call fails? Currently, nothing will be done, the call to `updateMFAConfiguration` will be stopped without any message or result.

Reply:
I will add validation for this as well. thank you